package.json

```diff
@@ -88,7 +88,7 @@
     "prettier": "1.19.1",
     "react": "16.10.2",
     "react-dom": "16.10.2",
-    "react-ssr-prepass": "1.0.7",
+    "react-ssr-prepass": "1.0.8",
```

Comment:
This can be removed now right?

Reply:
I thought so too but there's another test for serverless, so I added it back, no harm in having that test.